odoo/models.py

```diff
@@ -1767,15 +1767,19 @@ def get_formview_action(self, access_uid=None):
             'context': dict(self._context),
         }
 
-    def get_access_action(self, access_uid=None):
+    def _get_access_action(self, access_uid=None, force_website=False):
```

Comment:
Hmm, feels a bit strange to have a "force_website" parameter at models level, since there is no website nor portal module at this point.

Reply:
I agree, from modularity point of view this is thinking wiht website in mind. Somehow I find it better than a context key (which I dislike), as I don't have much other possibilities. If you have a suggestion feel free :) .

Reply:
We could do kwargs but it's less readable, especially if we only support one key, so I don't have anything better no.